### PR TITLE
Full URL notification

### PR DESCRIPTION
**Pull request by Mussuranae**
Change relative url on invitation and notification to full url (cross app problem)
Part of #5411

### CODE REVIEW

File: libs/invitation/src/lib/components/item/item.component.ts [hunk 1]
```diff
@@ -52,24 +55,25 @@ export class ItemComponent {
...
         }
         else {
-          return [`/c/o/marketplace/event`, this._invitation.eventId, 'session'];
+          return `${this.applicationUrl.festival}/c/o/marketplace/event/${this._invitation.eventId}/session`;
         }
       }
     } else if (this._invitation.type === 'joinOrganization') {
       const orgId = this._invitation.fromOrg ? this._invitation.fromOrg.id : this._invitation.toOrg.id;
-      return `/c/o/organization/${orgId}/view/members`;
+      return `${this.applicationUrl[this.app]}/c/o/organization/${orgId}/view/members`;
```
Comment (phpgeekfr) [inline libs/invitation/src/lib/components/item/item.component.ts:76]: so now, even if invitation is not for an event, we have the complete url ? Does it make the entire page reload if we click on an absolute url but we stay on same url  ?

**Review by phpgeekfr — APPROVED** [review]
LGTM

File: libs/invitation/src/lib/components/item/item.component.ts (hunk 1 above)
Comment (Mussuranae) [reply libs/invitation/src/lib/components/item/item.component.ts:76]: Yes it reload the page and redirect you to the new url

File: libs/utils/src/lib/apps.ts [hunk 2]
```diff
@@ -130,6 +132,18 @@ export function getOrgAppAccess(org: OrganizationDocument | Organization, first:
   }
 }
 
+/** Return an array of the app access of the movie */
+export function getMovieAppAccess(movie: MovieDocument | Movie): App[] {
+  const apps: App[] = [];
+  for(const a of app) {
+    const hasAccess = app.some(a => !!movie.storeConfig.appAccess[a]);
```
Comment (phpgeekfr) [inline libs/utils/src/lib/apps.ts]: shadowed variable `a`

**Pull request merged**